app/src/androidTest/java/com/google/android/perftesting/testrules/EnablePostTestDumpsys.java

```diff
@@ -47,6 +48,7 @@
 
     private String mTestName;
     private String mTestClass;
+    private double threshold;
```

Comment:
As per our discussion, rename this test rule to `MeasureGraphicStats` and use a more specific name for the threshold.